### ISSUE TITLE
Unused argument at function "nnsvs.gen.predict_duration"

### ISSUE THREAD

**Issue by oatsu-gh** [r9y9/nnsvs#59]
The argument `lag` looks unused in the function `nnsvs.gen.predict_duration`.

Whether should the state be removed or held for future development (for example duration prediction using timelag values)?
https://github.com/r9y9/nnsvs/blob/c69566f32c3799f9c0c0a76cc1f48175be9c653a/nnsvs/gen.py#L166-L167

original code follows
```python
def predict_duration(device, labels, duration_model, duration_config, duration_in_scaler, duration_out_scaler,
        lag, binary_dict, continuous_dict, pitch_indices=None, log_f0_conditioning=True):
    # Extract musical/linguistic features
    duration_linguistic_features = fe.linguistic_features(
        labels, binary_dict, continuous_dict,
        add_frame_features=False, subphone_features=None).astype(np.float32)

    if log_f0_conditioning:
        for idx in pitch_indices:
            duration_linguistic_features[:, idx] = interp1d(
                _midi_to_hz(duration_linguistic_features, idx, log_f0_conditioning),
                    kind="slinear")

    # Apply normalization
    duration_linguistic_features = duration_in_scaler.transform(duration_linguistic_features)
    if isinstance(duration_in_scaler, MinMaxScaler):
        # clip to feature range
        duration_linguistic_features = np.clip(
            duration_linguistic_features, duration_in_scaler.feature_range[0],
            duration_in_scaler.feature_range[1])

    # Apply model
    x = torch.from_numpy(duration_linguistic_features).float().to(device)
    x = x.view(1, -1, x.size(-1))

    if duration_model.prediction_type() == PredictionType.PROBABILISTIC:
        # (B, T, D_out)
        max_mu, max_sigma = duration_model.inference(x, [x.shape[1]])
        if np.any(duration_config.has_dynamic_features):
            # Apply denormalization
            # (B, T, D_out) -> (T, D_out)
            max_sigma_sq = max_sigma.squeeze(0).cpu().data.numpy() ** 2 * duration_out_scaler.var_
            max_mu = duration_out_scaler.inverse_transform(max_mu.squeeze(0).cpu().data.numpy())

            # (T, D_out) -> (T, static_dim)
            pred_durations = multi_stream_mlpg(max_mu, max_sigma_sq, get_windows(duration_config.num_windows),
                                              duration_config.stream_sizes, duration_config.has_dynamic_features)
        else:
            # Apply denormalization
            pred_durations = duration_out_scaler.inverse_transform(max_mu.squeeze(0).cpu().data.numpy())
    else:
        # (T, D_out)
        pred_durations = duration_model.inference(x, [x.shape[1]]).squeeze(0).cpu().data.numpy()
        # Apply denormalization
        pred_durations = duration_out_scaler.inverse_transform(pred_durations)
        if np.any(duration_config.has_dynamic_features):
            # (T, D_out) -> (T, static_dim)
            pred_durations = multi_stream_mlpg(
                pred_durations, duration_out_scaler.var_, get_windows(duration_config.num_windows),
                duration_config.stream_sizes, duration_config.has_dynamic_features)

    pred_durations[pred_durations <= 0] = 1
    pred_durations = np.round(pred_durations)

    return pred_durations
```

**Comment by r9y9** [comment]
You are right. I was originally thinking that phoneme durations may be predicted depending on the `lag` (or adjusted as in the `postprocess_duration` function). Since it is not used until now, removing the `lag` parameter from the `predict_duration` function should be okay.

**Comment by r9y9** [comment]
https://github.com/r9y9/nnsvs/pull/81/commits/43aead7e45d260ca2d1704ebbfdae55d64fa0f9a Fixed this in #81
